.github/actions/upload_terraform_module/action.yml

```
@@ -9,15 +14,23 @@ runs:
         cp -r terraform terraform-module
         find terraform-module -name "*.go" -type f -delete
         find terraform-module -name "*.bazel" -type f -delete
+
+    - name: Stamp version
+      shell: bash
+      run: |
+        sed -i "s/@@CONSTELLATION_VERSION@@/${{ inputs.version }}/g" terraform-module/constellation-cluster/main.tf
```

on Mac only this worked: `sed -i '' 's/@@CONSTELLATION_VERSION@@/v2/g' terraform/constellation-cluster/variables.tf`
I suggest to doublecheck this works

```suggestion
        sed -i "s/@@CONSTELLATION_VERSION@@/${{ inputs.version }}/g" terraform-module/constellation-cluster/variables.tf
```